nautobot/docs/development/application-registry.md

```diff
@@ -57,6 +57,73 @@ A dictionary of particular features (e.g. custom fields) mapped to the Nautobot
 }
 ```
 
+### `nav_menu`
+
+Navigation menu items provided by Nautobot applications. Each app registers its navbar configuration inside of the `nav_menu` dictionary. Tabs are stored in the top level moving down to groups, items and buttons. Tabs, groups and items can be modified by using the key values inside other application and plugins.
```

Comment:
Is `Each app must register` or `Each app may register` more accurate? 

tbh, reading the two updates to mark down document, it does not jump out at me how this works. Is this meant to go in navigation.py? Is this in lieu or in addition to the prior pattern?

Reply:
Will update with must. This is currently just for core nautobot, when this is made avaiable for plugins a more comprehensive guide will be created.

Reply:
I think it is a "may" - an app that has no need for menu items (such as `nautobot.utilities`) can skip defining a `nav_menu` module, correct?

Reply:
ok, thanks, I did not realize this was only for core. When I read registry, I thought it was for plugins to register.